frontend/src/components/pages/student/AssessmentSummaryPage.tsx

```diff
@@ -27,63 +28,41 @@ const AssessmentSummaryPage = (): React.ReactElement => {
   };
 
   return (
-    <HStack alignItems="flex-start" pt="4em">
-      <Image
-        alt="Jump Math Logo"
-        mx="2em"
-        src={JUMP_MATH_LOGO.src}
-        width="15%"
-      />
-      <VStack height="85vh" justifyContent="space-between" pr="4em">
-        <VStack align="left">
-          <Text color="blue.300" textStyle="header4">
-            {assessmentMetadata.testName}
-          </Text>
-          <Text color="blue.300" textStyle="paragraph">
-            Start Time: {assessmentMetadata.startDate} at{" "}
-            {assessmentMetadata.startTime}
-          </Text>
-          <SimpleGrid columns={{ base: 1, md: 2 }} gap={14} pt="3em">
-            <QuestionSummary
-              pointCount={assessmentMetadata.totalPoints}
-              questionCount={assessmentMetadata.numOfQuestions}
-            >
-              <VStack align="left" width="100%">
-                <br />
-                <Text paddingBottom="2" textStyle="smaller-paragraph">
-                  Question Types:
-                </Text>
-                <QuestionTypeImages
-                  questionTypes={assessmentMetadata.questionTypes}
-                />
-              </VStack>
-            </QuestionSummary>
-            <Box
-              backgroundColor="rgba(232, 237, 241, 0.2)"
-              borderRadius="10px"
-              padding="2em"
-            >
-              <Text color="blue.300" marginBottom="14px" textStyle="subtitle2">
-                Rules
+    <>
+      {test && (
+        <HStack alignItems="flex-start" pt="4em">
+          <Image
+            alt="Jump Math Logo"
+            mx="2em"
+            src={JUMP_MATH_LOGO.src}
+            width="15%"
+          />
+          <VStack height="85vh" justifyContent="space-between" pr="4em">
+            <VStack align="left">
+              <Text color="blue.300" textStyle="header4">
+                {test.name}
+              </Text>
+              <Text color="blue.300" textStyle="paragraph">
+                {/* update after test session backend is updated */}
+                Start Time: {assessmentMetadata.startDate} at{" "}
+                {assessmentMetadata.startTime}
               </Text>
-              <Stack gap={3}>
-                {assessmentMetadata.rules.split("\n").map((line, index) => (
-                  <Text key={index} textStyle="paragraph">
-                    {line}
-                  </Text>
-                ))}
-              </Stack>
-            </Box>
-          </SimpleGrid>
-        </VStack>
-        <HStack justifyContent="flex-end" spacing="1%" width="100%">
-          <Button onClick={handleBackToHome} variant="secondary">
-            Back to Home
-          </Button>
-          <Button variant="primary">Start Test</Button>
+              <SimpleGrid columns={{ base: 1, md: 2 }} gap={14} pt="3em">
+                <AssessmentInfo questions={test.questions} />
+                {/* update after test session backend is updated */}
```

Comment:
pending: https://www.notion.so/uwblueprintexecs/Update-Test-Session-Model-to-support-Distributing-Assessments-19cb55d738ed43e092cd48c1b4980675?pvs=4